hack/generate/samples/internal/go/memcached-with-customization/memcached_with_customization.go

```diff
@@ -134,7 +134,6 @@ func (mh *Memcached) Run() {
 		"--group", mh.ctx.Group,
 		"--version", mh.ctx.Version,
 		"--kind", mh.ctx.Kind,
-		"--defaulting",
 		"--defaulting")
```

Comment:
@acornett21, it was duplicated. If we want it in the sample, default + validating, then we need to add it here.
But would not be call twice `defaulting `
I keep only defaulting here. We can add the validating as well in a follow up if we need to